### ISSUE TITLE
Update for wp 4.3

### ISSUE THREAD

**Issue by bueltge** [bueltge/WordPress-Admin-Style#31]
https://make.wordpress.org/core/2015/07/08/today-in-the-nightly-customize-in-the-toolbar-passwords-ui-list-tables-on-phones-dashicons/

**Comment by bueltge** [comment]
Dashicons, New icons:

.dashicons-admin-customizer (f540)
.dashicons-admin-multisite (f541)
.dashicons-editor-table (f535)
.dashicons-filter (f536)
.dashicons-hidden (f530)
.dashicons-image-filter (f533)
.dashicons-image-rotate (f531)
.dashicons-layout (f538)
.dashicons-sticky (f537)
.dashicons-thumbs-down (f542)
.dashicons-thumbs-up (f529)
.dashicons-unlock (f528)
.dashicons-warning (f534)

**Comment by lordspace** [comment]
Great Job Frank. Thanks for continuing to work on the project.

This should the recommended approach when developing WP plugins as it would make the plugin UI so consistent with the WP's.

As soon as I discovered your project I've used the snippets in all my 80+ plugins :)
I will be giving a talk in a Sofia WordPress meetup and will be mentioning your cool project.
Not sure if it would be recorded though.

Slavi
